src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -1,20 +1,38 @@
 import React from "react";
+import { Button } from "react-bootstrap";
 import { PLAYA_VENUE_NAME } from "settings";
 import "./RadioModal.scss";
 
 export interface RadioModalPropsType {
   volume: number;
   setVolume: Function;
   title?: string;
+  onEnableHandler: () => void;
+  isRadioPlaying: boolean;
 }
 
 export const RadioModal: React.FunctionComponent<RadioModalPropsType> = ({
   volume,
   setVolume,
   title,
+  onEnableHandler,
+  isRadioPlaying,
 }) => {
-  return (
-    <div className="radio-modal-container">
+  const handleEnableButtonClick = () => {
+    onEnableHandler();
+    // Volume is randomized because if it is set to the same number as before
+    // the radio will not start playing
+    const volume = 20 + Math.random(); // i.e. 20 + 0.3
```

Comment:
That... seems strange. Do we know why that is?

Reply:
Leftover hack from previous integration if the volume was "set" to the previous value, the radio would not play.
Removing in new commit